Microsoft.Quantum.Katas/Utils.qs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+//////////////////////////////////////////////////////////////////////
+// This file contains some Q# intrinsic operations
+// needed for the validation of some Katas' answers.
+//////////////////////////////////////////////////////////////////////
+
+namespace Quantum.Kata.Utils {
```

Comment:
If I understand correctly, this namespace isn't part of the katas themselves (that is, doesn't include code written by users during a kata)? If so, I think that the consensus has been that it needs to live under the `Microsoft.Quantum.` prefix.
```suggestion
namespace Microsoft.Quantum.Kata.Utils {
```

Reply:
This would still be a breaking change. I'd rather not do it.